catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/SegmentHandler.java

```diff
@@ -104,6 +112,29 @@ private void handleTreValues(Metacard metacard, NitfAttribute attribute, Tre tre
   private Attribute populateAttribute(Metacard metacard, String attributeName, Serializable value) {
     Attribute currentAttribute = metacard.getAttribute(attributeName);
 
+    // check if country code
```

Comment:
❓ Instead of repeating L128-134 and L138-144, could we create a private method like `private List<Serializable> processAttributeValue(String attributeName, Serializable value)` that creates the attribute value(s), and have a single if/else block afterwards that knows what to do based on the existence of `currentAttribute` in the metacard regardless of the values?